src/modules/Instabug.ts

```diff
@@ -580,3 +580,20 @@ export const componentDidAppearListener = (event: ComponentDidAppearEvent) => {
     _lastScreen = event.componentName;
   }
 };
+
+/**
+ * Sets listener to W3ExternalTraceID flag changes
+ * @param handler A callback that gets the update value of the flag
+ */
+export const _registerW3CFlagsChangeListener = (
```

Comment:
Let's annotate this with `@internal`. We can also add the `stripInternal` TSConfig flag as `true` to strip `@internal` functions like this from the type definitions, what do you think?

```suggestion
/**
 * Sets listener to W3ExternalTraceID flag changes
 * @param handler A callback that gets the update value of the flag
 * @internal
 */
export const _registerW3CFlagsChangeListener = (
```